### ISSUE TITLE
Is it possible to close the image picker after the user selects the first image?

### ISSUE THREAD

**Issue by mikepc** [wymsee/cordova-imagePicker#164]
When selecting a profile picture for instance, I have set maximumImagesCount to 1, but the user can select multiple images. It just uses one of them, but it's confusing to the user.

**Comment by sava999** [comment]
@mikepc Use [camera plugin](https://github.com/apache/cordova-plugin-camera) with [source type](https://github.com/apache/cordova-plugin-camera#camerapicturesourcetype--enum) set to photo library.

**Comment by Goosaroo** [comment]
yes, just set the maximum number of pictures to one (maximumImagesCount: 1). It will directly close the ui after selecting one image.

**Comment by Goosaroo** [comment]
Weird, I use the same setting with maximumImagesCount and it directly closes the UI after selecting just one image.
